<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ПАО Сбербанк</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>901</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>download</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>751</width>
       <height>71</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>9</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>       с</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>221</x>
        <y>9</y>
        <width>27</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>     по</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="tm223fzTendBeg">
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>9</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>3</month>
        <day>13</day>
       </datetime>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkTand223fz">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>59</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>223-ФЗ</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="tm223fzTendEnd">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>9</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>1</month>
        <day>17</day>
       </datetime>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>инн</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>7707083893</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dirTand223fz">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>G:/toSber/downloadToDisk/debug/new</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>куда</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btn223fzTendPath">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTmTend223fz">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>7</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTmTend223fz_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>7</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn223fzNotifRegions">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>регионы</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>751</width>
       <height>71</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>9</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>       с</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>221</x>
        <y>9</y>
        <width>27</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>     по</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="tm44fzTendBeg">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>9</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>3</month>
        <day>13</day>
       </datetime>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkTand44fz">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>59</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>44-ФЗ</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="tm44fzTendEnd">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>9</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>инн</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>7707083893</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dirTand44fz">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>куда</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btn44fzTendPath">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTmTend44fz_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>7</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTmTend44fz">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>7</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>751</width>
       <height>71</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>9</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>       с</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>221</x>
        <y>9</y>
        <width>27</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>     по</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="tm223fzBeg">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>9</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>3</month>
        <day>13</day>
       </datetime>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkDis223fz">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>59</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>223-ФЗ</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="tm223fzEnd">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>9</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>3</month>
        <day>17</day>
       </datetime>
      </property>
     </widget>
     <widget class="QLineEdit" name="dir223fz">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>куда</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btn223fzPath">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTm223fz">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>7</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTm223fz_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>7</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn223fzDishonRegions">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>регионы</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>недобросовестные поставщики</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>751</width>
       <height>71</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>9</y>
        <width>30</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>       с</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>221</x>
        <y>9</y>
        <width>27</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>     по</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="tm44fzBeg">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>9</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkDis44fz">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>59</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>44-ФЗ</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="tm44fzEnd">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>9</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="dir44fz">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>471</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>куда</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btn44fzPath">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>25</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTm44fz">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>7</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTm44fz_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>7</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>тендеры</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
